Hamburger Menu done,

JanikOskar opened the pull request:
hamburger menu details are visible and styled accordingly to designs

active menu should be selected

on click redirection to /my-personas, /my-spots

logout functionality

background-color should be grayed

JanikOskar committed:
Hamburger Menu done,
hamburger menu details are visible and styled accordingly to designs

active menu should be selected

on click redirection to /my-personas, /my-spots

logout functionality

background-color should be grayed

lmazurowski commented on src/global/Layouts/HamburgerMenu/HamburgerMenu.tsx:
```diff
@@ -0,0 +1,179 @@
+      {isWithHamburger && (
+        <HamburgerMenuStyled>
+          <HamburgerMenuIcon color={themeContext.colors.main.tetiary} onClick={() => setIsMenuOpened(true)} />
+          <StyledDrawer placement="left" closable={false} onClose={() => setIsMenuOpened(false)} visible={isMenuOpened}>
```
Ok, so the size of this component can be reduce by extracting StyledDrawer together with MenuDivider and DrawerWrapper. You can have component called Drawer so that it can be build like this:
`<StyledDrawer>
<HeaderDivider/>
<DrawerWrapper>
{children}
</DrawerWrapper>
</StyledDrawer>
`

lmazurowski commented on src/global/Layouts/HamburgerMenu/HamburgerMenu.tsx:
```diff
@@ -0,0 +1,179 @@
+            </DrawerWrapper>
+          </StyledDrawer>
+          {isWithSearch && (
+            <SearchPositionBox>
```
SearchPositionBox can be also extracted outside in the same folder.

lmazurowski commented on src/global/Layouts/HamburgerMenu/common.tsx:
```diff
@@ -0,0 +1,11 @@
+}
+
+// this prevents one console error
+export const LinkWithoutIsActiveInDom = ({ isActive, children, ...restProps }: StyledLinkProps & LinkProps) => (
```
Where are You using isActive?
I deleted this component its unnecessary

lmazurowski commented on src/global/graphqls/SignOut.ts:
```diff
@@ -0,0 +1,13 @@
+import { gql } from 'apollo-boost';
+
+export interface SignOutResponse {
```
I think this is wrong response , right? Please check the response in the "Network" tab when You clicked on the logout. It's somethink like logout: boolean

JanikOskar committed:
Drawer as different component
Hamburger With Search as different component
SignOutResponse change
common deleted(unnecessary file)

laniszew commented on src/components/Drawer/Drawer.tsx:
```diff
@@ -0,0 +1,53 @@
+import { Icon } from '../Icon';
+import HamburgerMenuSvg from 'assets/HamburgerMenu.svg';
+
+const StyledDrawer = styled(Drawer).attrs({
```
isn't using .attrs outdated? Isn't there a console.warn about that?
